src/modules/users/users.query.js

```diff
@@ -3,20 +3,41 @@ const createUserQuery = (body) => {
         CREATE (u: Student 
             {
                 uuid: "${body.uuid}", 
-                totalExp: "0", 
-                weeklyExp: "0", 
+                totalExp: 0, 
+                weeklyExp: 0, 
                 email: "${body.email}", 
-                userName: "${body.userName}"
+                userName: "${body.userName}", 
+                password: "${body.password}"
             }
-        )
+        );
+    `;
+
+    return query;
+};
+
+const deleteUserQuery = (body) => `MATCH (u: Student {uuid: "${body.uuid}"}) DELETE u;`;
+const findRegisteredUser = (body) => `MATCH (u: Student {email: "${body.email}"}) RETURN u;`;
+const findUserQuery = (params) => `MATCH (u: Student {uuid: "${params.uuid}"}) RETURN u;`;
+
+const updateUserQuery = (body) => {
+    const query = `
+        MATCH (u: Student {uuid: "${body.uuid}"})
```

Comment:
There are two options here one we could pass in an options menu 
OR 
We could query the db for the user and .then when we have the info of the user we can see the props of body and see what props they want to change. This way we could just do a ternary operator when building the query.